source/nodejs/adaptivecards/src/enums.ts

```diff
@@ -152,7 +152,8 @@ export enum ValidationError {
     TooManyActions,
     UnknownActionType,
     UnknownElementType,
-    UnsupportedCardVersion
+    UnsupportedCardVersion,
+    DuplicateId
```

Comment:
Hmm it's too late now, but precisely for this reason (to be able to extend the enum) generally prefer to start it off wiht a default/UnsupportedCardVersion enum value rather than having it last :( ) If we'd started out with Unknown or Unsupported, we could have continued to extend the enum without worrying about breaking the flow of the type definition

Reply:
Aah n/m -- i think i may be misinterpreting the UnsupportedCardVersion to read as the enum default which it isnt ..